src/sql/parse.py

```diff
@@ -65,6 +66,18 @@ def parse(cell, config):
         if len(pieces) == 2:
             return result
         cell = pieces[2]
+    # deal with situation var =<< 'sql'
+    elif len(pieces) > 1 and pieces[1] == "=<<":
+        result["result_var"] = pieces[0]
+        result["return_result_var"] = True
+        cell = pieces[2]
+
+    # deal with situation var = << 'sql'
```

Comment:
`deal with situation var = << 'sql'` -> `handle flexible spacing around =`

Reply:
Update it now